packages/lesswrong/components/messaging/MessagesNewForm.tsx

```diff
@@ -66,15 +100,16 @@ export const NewMessageForm = ({
   });
 
   const SubmitComponent = ({ submitLabel = "Submit" }) => {
+    const formButtonClass = isMinimalist ? classes.formButtonMinimalist : classes.formButton
```

Comment:
Not a change made in this PR, but `SubmitComponent` should be wrapped in `useCallback` (I also frequently forget to do this).